pkg/export/bench/.gitingore

```diff
@@ -0,0 +1,3 @@
+data1
```

Comment:
What are these? Can you add a comment? IIUC they're local TSDB dirs...

Reply:
Yes, TSDB dirs created when running the benchmarking script.